c7n/query.py

```diff
@@ -540,6 +529,54 @@ def generate_arn(self):
         return self._generate_arn
 
 
+class MaxResourceLimit(object):
+
+    def __init__(self, policy, selection_count, population_count):
+        self.p = policy
+        self.op = None
+        self.selection_count = selection_count
+        self.population_count = population_count
+        self.amount = selection_count
+        self.percentage_amount = population_count
+        self.percent = 100
+        self._parse_policy()
+
+    def _parse_policy(self,):
+        if isinstance(self.p.max_resources, dict):
+            self.op = self.p.max_resources.get("op", "or").lower()
```

Comment:
let's get the default here from an environment variable, C7N_MAXRES_OP ? or C7N_CIRCUIT_OP ?